README.md

```diff
@@ -18,35 +18,26 @@ It also includes full support for iOS, Android & Windows.
 
 ### Running the KitchenSink App
 
-#### Via Appcelerator Studio
-
-1. Import it via *Dashboard* if available.
-2. Or import it via *File > Import... > Git > Git Repository as New Project* with *URI*:
-
-		https://github.com/appcelerator/kitchensink-v2
-
-3. Select a Simulator or Device to build to via *Run > Run As*.
-
 #### Via CLI
 
-1. Clone the repository:
+1.  Clone the repository:
 
-		git clone https://github.com/appcelerator/kitchensink-v2
+        git clone https://github.com/tidev/kitchensink-v2
 
-2. To run it with `appc run` first import it to the platform:
+2.  To run it with `ti build` first import it to the platform:
 
-		appc new --import --no-services
+        ti create --import --no-services
 
-3. Build to Simulator or Device:
+3.  Build to Simulator or Device:
 
-		[appc run | ti build] -p ios [-T device]
+        ti build -p ios [-T device]
 
 ### Contribution
 
-Code contributions are greatly appreciated, please submit a new [pull request](https://github.com/appcelerator/kitchensink-v2/pull/new/master)!
+Code contributions are greatly appreciated, please submit a new [pull request](https://github.com/tidev/kitchensink-v2/pull/new/master)!
 
-### Legal Stuff
+<!-- ### Legal Stuff
 
 Appcelerator is a registered trademark of Appcelerator, Inc. Titanium is
-a registered trademark of Appcelerator, Inc.  Please see the LEGAL information about using our trademarks,
-privacy policy, terms of usage and other legal information at [http://www.appcelerator.com/legal](http://www.appcelerator.com/legal).
+a registered trademark of Appcelerator, Inc. Please see the LEGAL information about using our trademarks,
+privacy policy, terms of usage and other legal information at [http://www.appcelerator.com/legal](http://www.appcelerator.com/legal). -->
```

Comment:
The legal link is dead. I can't find an analog under axway.com. This whole legal stuff section could probably be removed.

Reply:
removed now